deps/wazuh_testing/wazuh_testing/tools/utils.py

```diff
@@ -115,6 +115,17 @@ def get_random_ip():
     return fr"{randint(0,255)}.{randint(0,255)}.{randint(0,255)}.{randint(0,255)}"
 
 
+def get_current_ip() -> str:
+    """Retrieves the ip address of this machine.
+
+    Return:
+        String: Machine's IP address.
+    """
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    s.connect(("8.8.8.8", 80))
+    return s.getsockname()[0]
```

Comment:
The hardcoded IP is because it is just needed to do a ping to that ip (a google dns) I dont think we need to receive an IP from params or having it stored somewhere else.
I didnt find a method that does the same, where is it?

Reply:
Comments added on commit [`a915da5`](https://github.com/wazuh/wazuh-qa/pull/3662/commits/a915da5cf02906ea8e70673489fb69f3bde08639)